packages/react/src/views/AttachmentHandler/TextAttachment.js

```diff
@@ -12,11 +12,6 @@ const TextAttachment = ({ attachment, type, variantStyles = {} }) => {
     return URL;
   };
 
-  let attachmentText = attachment?.text;
```

Comment:
Why you removed this.. this is needed when you try to render a instruction message sent by RC app